### DISCUSSION TITLE
What is debug mode?

### DISCUSSION THREAD

**Discussion by bitjerry** [localsend/localsend#847] [Discussion (English, International)] [answered]
What is debug mode, how should I turn it on, and what are the options for app arguments? I have not found any documentation on this. Understanding these parameters is very helpful in troubleshooting problems.

![image](https://github.com/localsend/localsend/assets/52879142/5573655b-ab33-489b-800c-7ed429d0edc3)

**Comment by Tienisto** [comment, marked as answer]
This is a Flutter feature where the app runs in interpreted mode with hot reloading. The built binaries in the releases (and the app store versions) are always compiled in production mode.

This is just an info for the developers to know what version it is
